httpsender/add-more-headers.js

```diff
@@ -0,0 +1,64 @@
+// This HttpSender script adds headers to all messages transmitted by zaproxy,
+// including automated tools. Refer to the HttpSender class definition:
+// https://github.com/zaproxy/zaproxy/blob/master/zap/src/main/java/org/parosproxy/paros/network/HttpSender.java
+// for a list of 'initiator' values (although we don't use them).
+
+var ScriptVars = Java.type("org.zaproxy.zap.extension.script.ScriptVars");
+
+/*
+ * HttpSender scripts do not support parameters, so we'll use a known global
+ * variable to supply desired content. The value of this variable should be a
+ * JSON string containing a serialized map<String, String> object. The map keys
+ * are the desired header name and the values are the header values.
```

Comment:
Including an example would probably be a good move. I think I see what you're meaning but I'm not sure others will.
```json
{"X-Forward-For":"localhost","some-other-header":"some-value"}
```

Reply:
I added an example (and found a statement that was misleading).